roxie/ccd/ccdmain.cpp

```diff
@@ -1572,7 +1575,7 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
     UseSysLogForOperatorMessages(false);
     ExitModuleObjects();
     releaseAtoms();
-    strdup("Make sure leak checking is working");
+    leakChecker = strdup("Make sure leak checking is working");
```

Comment:
There are some other instances as well.  Line 1573 in the same fiile.  Also udpsim.cpp and udptest.cpp.

Should this be set to null later on?  (So valgrind will report it as definitely lost rather than reachable.)